models/grading/unit_tests.yaml

```diff
@@ -38,33 +38,3 @@ unit_tests:
     expect:
       format: csv
       fixture: fact_learner_course_status_expected
-
-  - name: test_fact_student_status
```

Comment:
Why did you removed this test?

Reply:
It fails with the 'final' keyword, not sure how to handle it yet so i just got rid of it for now